oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -226,9 +226,16 @@ public OSProcess getProcess(int pid) {
         long now = System.currentTimeMillis();
         OSProcess proc = new OSProcess();
         // See man proc for how to parse /proc/[pid]/stat
-        proc.setName(split[1].replaceFirst("\\(", "").replace(")", ""));
+        int nameOffset = 0;
+        StringBuilder name = new StringBuilder(
+                split[1].replaceFirst("\\(", ""));
+        while (!split[1 + nameOffset].endsWith(")")) {
+            nameOffset++;
+            name.append(' ').append(split[1 + nameOffset]);
+        }
+        proc.setName(name.substring(0, name.length() - 1));

```

Comment:
This all looks good, other than the part of me that thinks we should do some sort of error checking.  While we can probably rely on an actual `/proc/pid/stat` result to be formatted as required, we should check a "just in case" it's an empty string.  Perhaps test the existence of the `(` character in the string?